getting_started/first_3d_game/04.mob_scene.rst

```diff
@@ -164,14 +164,17 @@ between ``-PI / 4`` radians and ``PI / 4`` radians.
         # In this rotation^, the mob will move directly towards the player
         # so we rotate it randomly within range of -90 and +90 degrees.
         rotate_y(randf_range(-PI / 4, PI / 4))
+
  .. code-tab:: csharp
 
-    // We will call this function from the Main scene
+    // This function will be called from the Main scene.
     public void Initialize(Vector3 startPosition, Vector3 playerPosition)
     {
-        // We position the mob and turn it so that it looks at the player.
+        // We position the mob by placing it at startPosition
+        // and rotate it towards playerPosition, so it looks at the player.
         LookAtFromPosition(startPosition, playerPosition, Vector3.Up);
-        // And rotate it randomly so it doesn't move exactly toward the player.
+        // In this rotation^, the mob will move directly towards the player
+        // so we rotate it randomly within range of -90 and +90 degrees.
```

Comment:
```suggestion
        // In this rotation, the mob will move directly towards the player
        // so we rotate it randomly within range of -90 and +90 degrees.
```
Maybe even `With this rotation` (not 100% sure if that is correct).

Or maybe reword this whole comment a bit:
```suggestion
        // Rotate this mob randomly within range of -90 and +90 degrees,
        // so that it doesn't move directly towards the player.
```

Reply:
The `^` is likely an arrow pointing to the previous line. I agree though that `with this rotation` sounds better.

I actually like your reworded comment more than what we currently have, but I'd like to also get the opinion of @TheYellowArchitect who originally wrote this comment in the GDScript version.

Reply:
Yeah the `^` is weird, can definitely be rephrased.

Reply:
Changed.